server/server/data_loader/fuzzymatcher.py

```diff
@@ -0,0 +1,68 @@
+from collections import defaultdict, Counter
+
+class FuzzyMatcher:
+    """
+    
+    Uses ngrams to identify similiar words
+    
+    You'd think this would be a solved problem but I looked into other
+    options on the internet and there are a lot but most of them
+    are either O(N^2) or just way too slow.
+    This method works surprisingly well for pali and takes mere seconds
+    to find similiarties within a 40000 word dictionary.
+    
+    """
+    
+    # Future Improvements that aren't a huge deal but would be nice:
+    # 1. Certain combinations like kh, dh etc are actually a single
+    #    letter and should be treated as such.
+    
+    def __init__(self, words, threshold=0.0):
```

Comment:
just threshold initialize as `0`.

Reply:
It's 0.0 because 0 is int and 0.0 is float, and it is meant to be a float. Also using 0.0 for floats seems consistent with the python standard library documentation.